include/mcl_3dl/point_cloud_random_sampler.h

```diff
@@ -30,18 +30,28 @@
 #ifndef MCL_3DL_POINT_CLOUD_RANDOM_SAMPLER_H
 #define MCL_3DL_POINT_CLOUD_RANDOM_SAMPLER_H
 
+#include <vector>
+
 #include <pcl/point_cloud.h>
 
 #include <mcl_3dl/chunked_kdtree.h>
+#include <mcl_3dl/state_6dof.h>
 
 namespace mcl_3dl
 {
 template <class POINT_TYPE>
 class PointCloudRandomSampler
 {
 public:
+  virtual void loadConfig(const ros::NodeHandle& nh)
```

Comment:
ROS header should be included.

BTW, it would be better to refactor them to use custom struct instead of directly loading from ROS parameters for testability and reusability. (not related to this PR)